source/functions/BEMIO/readCAPYTAINE.m

```diff
@@ -202,7 +202,7 @@
         if hydro(F).Nb == 1
             fileID = fopen(fullfile(meshdir,'KH.dat'));
         else
-            fileID = fopen([fullfile(meshdir,'KH_'),num2str(m-1),'.dat']);
+            fileID = fopen([fullfile(meshdir,'KH_'),num2str(m),'.dat']);
```

Comment:
I think this should be left as m-1 to keep with existing convention of starting from KH_0.dat, Hydrostatics_0.dat, etc